### PR TITLE
[Fedora] Remove extra systemd services from the container

### PR DESCRIPTION
**Pull request by black-dragon74**
Signed-off-by: black-dragon74 <nickk.2974@gmail.com>

### CODE REVIEW

File: Fedora/Dockerfile [hunk 1]
```diff
@@ -33,6 +33,14 @@ RUN dnf -y update && \
     sed -i "s/LANG/\#LANG/g" /etc/locale.conf && \
     dnf -y install systemd-udev glusterfs-server python3-dbus nfs-utils attr iputils iproute glusterfs-geo-replication openssh-server openssh-clients cronie tar rsync sos xfsprogs && \
     dnf clean all && \
+    (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done) && \
```
Comment (nixpanic) [inline Fedora/Dockerfile:36]: deleting files from `/lib` is really not clean, better to disabled these with a systemctl command as done further below.

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: Fedora/Dockerfile (hunk 1 above)
Comment (black-dragon74) [reply Fedora/Dockerfile:36]: I guess I should apply the same to `CentOS` container then?

Comment (nixpanic) [reply Fedora/Dockerfile:36]: yes, that would be the cleanest

Comment (black-dragon74) [reply Fedora/Dockerfile:36]: It makes sense to close this PR. The same issue is rectified for CentOS container at #196 

Regards

Comment (black-dragon74) [reply Fedora/Dockerfile:36]: @nixpanic It is a standard practice wherever systemd is used inside docker to run these commands to stop extra services from running. One such example is [here](https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container).

I think it is safe to merge this and scratch #196 

Regards

Comment (nixpanic) [reply Fedora/Dockerfile:36]: I'm still not very fond of removing files from an installed RPM. It is bad practise in almost all environments, except for container-images. Disabling/masking the services with the `systemctl` command would be much cleaner, and sticks with the same prpcess that should be applied on other environments.

That said, container-images are an exception... So, it is not a blocking issue, just something that could be done cleaner.

**Review by nixpanic — APPROVED** [review]

**Pull request merged**